src/util/triplea/MapXMLCreator/MapXMLHelper.java

```diff
@@ -0,0 +1,1008 @@
+package util.triplea.MapXMLCreator;
+
+import games.strategy.engine.data.GameParser;
+import games.strategy.util.Triple;
+import games.strategy.util.Tuple;
+
+import java.awt.HeadlessException;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FilenameFilter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.LinkedHashMap;
+import java.util.LinkedHashSet;
+import java.util.Map.Entry;
+import java.util.SortedSet;
+import java.util.TreeSet;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.transform.OutputKeys;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.dom.DOMSource;
+import javax.xml.transform.stream.StreamResult;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.NamedNodeMap;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+import org.xml.sax.SAXException;
+
+import util.image.FileOpen;
+import util.image.FileSave;
+import util.triplea.MapXMLCreator.TerritoryDefinitionDialog.DEFINITION;
+
+import com.sun.org.apache.xerces.internal.impl.xpath.regex.ParseException;
+
+/**
+ * This class reads, writes and keeps the Map XML properties.
+ * 
+ * @author Erik von der Osten
+ * 
+ */
+public class MapXMLHelper {
+
+	private static final String TRIPLEA_JAVA_CLASS_DELEGATE_PATH = "games.strategy.triplea.delegate.";
+	static LinkedHashMap<String, String> s_xmlStrings = new LinkedHashMap<String, String>();

```

Comment:
This class is just carrying those variable. Do you really think we need another wrapper method for each access (get, contains, etc.)? I already need to add around 150 lines just for clear and add/put methods.
The reason why encapsulation is not enforced so strongly here is due to the fact that all those variables are basically the XML inside the application and the different panels are accessing/changing one or more of them at the time.
If you think hard encapsulation is still needed, I would add a TODO as it is a longer task with all the dependencies.